scripts/test-runner.sh

```diff
@@ -47,8 +47,13 @@ if [[ $OS_TYPE != "fedora" ]]; then
     $PKG_MAN -y install epel-release
 fi
 
+# TEMPORARY WORKAROUND
+# TODO: Remove when a fixed official beakerlib version is released
+yum -y install https://sumsal.cz/dev/beakerlib-1.11centos-99.fc23.noarch.rpm
```

Comment:
@mrc0mmand I've just communicated with our beakerlib maintainer (dapospis) and he told me the -a argument was added there to address https://bugzilla.redhat.com/show_bug.cgi?id=1388422 . If you would be able to come up with a fix that satisfies needs of both the mentioned bugzilla and yours, we can get it to testing beakerlib in a matter of hours.